_tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api.md

```diff
@@ -0,0 +1,207 @@
+|`consecutive_failure_count`	| The number of consecutive remote refresh failures since the last success.	|
+|`total_remote_refresh`	|The total number of remote refreshes.	|
+|`total_uploads_in_bytes`	| The total number of bytes in all uploads to the remote store. |
+|`remote_refresh_size_in_bytes.last_successful`	|The size of data uploaded in the last successful refresh.	|
+|`remote_refresh_size_in_bytes.moving_avg`	|The average size of data (in bytes) uploaded in the last _N_ refreshes. _N_ is defined in `remote_store.segment.pressure.upload_bytes_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).	|
+|`upload_latency_in_bytes_per_sec.moving_avg`	|The average speed of remote store uploads (in bytes per second) for the last _N_ uploads. _N_ is defined in `remote_store.segment.pressure.upload_bytes_per_sec_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).		|
+|`remote_refresh_latency_in_millis.moving_avg`	|The average time taken by a single remote refresh during the last _N_ remote refreshes. _N_ is defined in `remote_store.segment.pressure.upload_time_moving_average_window_size`. For details, see [Remote segment backpressure]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-segment-backpressure/).	|
+
+## Single shard stats
+
+Use the following API to get remote store statistics for a single shard.
+
+#### Example request
+
+```json
+GET _remotestore/stats/<index_name>/<shard_id>
+```
+{% include copy-curl.html %}
+
+#### Example response
+
+<details open markdown="block">
+  <summary>
+    Response
+  </summary>
+  {: .text-delta }
+
+```json
+{
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "failed": 0
+  },
+  "stats": [
+    {
+      "shard_id": "[so][0]",
+      "refresh_time_lag_in_millis": 5727,
+      "refresh_lag": 1,
+      "bytes_lag": 0,
+      "backpressure_rejection_count": 0,
+      "consecutive_failure_count": 0,
+      "total_remote_refresh": {
+        "started": 57,
+        "succeeded": 56,
+        "failed": 0
+      },
+      "total_uploads_in_bytes": {
+        "started": 1568138701,
+        "succeeded": 1568138701,
+        "failed": 0
+      },
+      "remote_refresh_size_in_bytes": {
+        "last_successful": 12705142,
+        "moving_avg": 32766119.75
+      },
+      "upload_latency_in_bytes_per_sec": {
+        "moving_avg": 25523682.95
+      },
+      "remote_refresh_latency_in_millis": {
+        "moving_avg": 990.55
+      }
+    }
+  ]
+}
+```
+</details>
+
+### Single local shard stats
+
+Provide the `local` query parameter set to `true` to only fetch the shards present on the node that is serving the request:
```

Comment:
```suggestion
Set the `local` query parameter to `true` to only fetch the shards present on the node that is serving the request:
```

Reply:
This parameter is optional so I think "provide" conveys that meaning.